dvc/scm/git/__init__.py

```diff
@@ -133,15 +132,24 @@ def _get_gitignore(self, path):
 
         return entry, gitignore
 
-    @staticmethod
-    def _ignored(entry, gitignore_path):
-        if os.path.exists(gitignore_path):
-            with open(gitignore_path, "r") as fobj:
-                ignore_list = fobj.readlines()
-            return any(
-                filter(lambda x: x.strip() == entry.strip(), ignore_list)
-            )
-        return False
+    def _ignored(self, entry, gitignore_path):
```

Comment:
You can simply pass `path` to it now, since we no longer work with entries. :slightly_smiling_face: Your PR is helping tidy this a lot :slightly_smiling_face: